### PR TITLE
Issue-#849: Update marketplace content

### LINKED ISSUE #849: Update Set Up a Marketplace content
**Is your feature request related to a problem? Please describe.**
Update docs for setting up own marketplace.

Remove existing content and Mention link: https://blog.oceanprotocol.com/launch-a-blockchain-based-data-marketplace-in-under-1-hour-9baa85a65ece
Add a new page related to Fees. -> Refer smart contract and explain how fees can be set. 

**Describe the solution you'd like**
Update content at: https://docs.oceanprotocol.com/tutorials/marketplace/

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #849

Changes proposed in this PR:

- [x] Update setup marketplace page
- [x] Add content on fees

### CODE REVIEW

**Comment by alexcos20** [comment]
@alexcos20  - pls add content

**Comment by AnaLoznianu** [comment]
@akshay-ap Looks good so far, let's please add the fees table and see how it looks like.

File: content/tutorials/marketplace-fees.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Fees
+description: The Ocean Protocol defines various fees for creating a sustainability loop.
+---
+
+## Path to sustainability
+
+Ocean Protocol aims to be a community-driven project. Through OceanDAO, the project aims at achieving decentralized governance and sustainability loop. For the project to grow and improve, the participants need to be rewarded for their contributions. The Ocean Protocol's smart contracts include a fee mechanism to maintain the sustainability loop.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:8]: Suggestions:
"aims to be" or "it is"? 
"need to be rewarded" or "are rewarded"?

File: content/tutorials/marketplace-fees.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Fee types
+
+### Swap fee
+
+Swap fees are collected whenever someone swaps a datatoken for basetoken (e.g., OCEAN) or basetoken for datatoken. The swap could be inside a pool (using an automated market maker) or in a fixed-rate exchange.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:14]: I know "datatoken" is a single word for various reasons but @trentmc have we decided to do the same for "basetoken"? Or should we have two words here?

File: content/tutorials/marketplace-fees.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+### Ocean Community fee
+
+Ocean's smart contract collects the **Ocean Community fee** during a swap operation. These fees are reinvested in community projects via Ocean DAO and other initiatives. Currently. This fee is set to 0 (as long as either Ocean or H20 are used as the base token in the market) in the smart contract but can be updated by the Ocean Protocol foundation.
+
+Whereas a fee of 0.1% is applicable on all swaps using another token as the base token.
+
+### Provider fee
+
+Provider is a component of Ocean Protocol's ecosystem which facilitates data consumption, starting compute jobs, encryption of DDOs, and decryption of DDOs. Provider also validates if the user can access a particular data asset or service. To know more about Provider, click here.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:49]: Suggestions: 
"starting compute jobs" -> "starts compute jobs". Same for encryption and decryption
"Provider also validates" -> "The provider also validates"

File: content/tutorials/marketplace-fees.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+The following fees apply to the consumption of data assets as a part of Consume fee:
+
+- Consume Market Consumption Fee
+- Publisher Market Consumption Fee
+- Provider Consumption Fees
+- [Ocean Community Fee](#ocean-community-fee)
+
+### Ocean Community fee
+
+Ocean's smart contract collects the **Ocean Community fee** during a swap operation. These fees are reinvested in community projects via Ocean DAO and other initiatives. Currently. This fee is set to 0 (as long as either Ocean or H20 are used as the base token in the market) in the smart contract but can be updated by the Ocean Protocol foundation.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:43]: " Currently. This fee is set to" -> " Currently, this fee is set to"

File: content/tutorials/marketplace-fees.md (hunk 3 above)
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:49]: You wanted to add a link at the end of the provider fee? Click here....but no link is attached

File: content/tutorials/marketplace-fees.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+Whereas a fee of 0.1% is applicable on all swaps using another token as the base token.
+
+### Provider fee
+
+Provider is a component of Ocean Protocol's ecosystem which facilitates data consumption, starting compute jobs, encryption of DDOs, and decryption of DDOs. Provider also validates if the user can access a particular data asset or service. To know more about Provider, click here.
+
+Provider fees are paid to the individual or organization running their Provider instance whenever a user consumes a data asset or swaps a datatoken for a basetoken. These fees can be set to an absolute amount, and the receiver can also decide which token to receive the fees in - they don't have to be in the same currency used in the consuming market.
+
+Provider fee can also be used to charge for computing resources. Based on the compute resources needed to run an algorithm in the Compute-to-Data environment, a consumer can choose the amount of pay according to their needs.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:53]: Suggestion: "the amount of pay" -> "the amount to pay"

File: content/tutorials/marketplace-fees.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+| --------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ |
+| publishMarket: Pools                                                                                                  | Set in market config, by OPF Value = 0 (agreed Feb 28 by T, A, Mihai, Matthias, Ana)                                                                                                                                      | Set in market config, by publishMarket                       |
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:61]: @akshay-ap Please remove all the references of "agreed on" and the members. This was relevant for us internally

Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:61]: Suggestion: "Set in market config, by publishMarket" --> "Set in the market config, by the publishing market"

File: content/tutorials/marketplace-fees.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+| publishMarket: FixedRate                                                                                              | Set in market config, by OPF Value = 0 (agreed etc.)                                                                                                                                                                      | SSet in market config, by publishMarket                      |
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:62]: Minor typo here on the "SSet", also the same suggestion as above: "Set in the market config, by the publishing market"

File: content/tutorials/marketplace-fees.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+Provider is a component of Ocean Protocol's ecosystem which facilitates data consumption, starting compute jobs, encryption of DDOs, and decryption of DDOs. Provider also validates if the user can access a particular data asset or service. To know more about Provider, click here.
+
+Provider fees are paid to the individual or organization running their Provider instance whenever a user consumes a data asset or swaps a datatoken for a basetoken. These fees can be set to an absolute amount, and the receiver can also decide which token to receive the fees in - they don't have to be in the same currency used in the consuming market.
+
+Provider fee can also be used to charge for computing resources. Based on the compute resources needed to run an algorithm in the Compute-to-Data environment, a consumer can choose the amount of pay according to their needs.
+
+This fee incentivizes individuals and organizations to run their provider instances and charge consumers according to resource usage.
+
+## Fee values
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace-fees.md:57]: As a general comment for the table, I suggest the following: 
On the second column(with OPF) values, let's make the values prominent and disregard the setting up.

For instance "Set in market config, by OPF Value = 0 (agreed Feb 28 by T, A, Mihai, Matthias, Ana)" it can be just "0".
Cause the details on how to set this are in the third column of the table, for people who want to run their marketplaces. 

"OPF gov can update in smart contract Value = 0.1% (agreed etc.)" --> "0.1%"

"Default 1%; Set by pool creator on contract deployment Min allowed (in contract) = 0.001% Min allowed (in Ocean Market) = 0.1% Max allowed (in contract) = 10% Min allowed (in Ocean Market) = 10%" ----> this would be "1%. and then the max and min values"

File: content/tutorials/marketplace.md [hunk 9]
```diff
@@ -3,43 +3,19 @@ title: Set Up a Marketplace
 description:
 ---
 
-In Ocean, marketplaces and publishers are different roles. A common setup is for one organization to do both. We focus on that here.
+## About marketplace
 
-## The Steps
+Ocean Protocol's [marketplace](https://market.oceanprotocol.com/) provider a web interface for accessing assets published on the chain. By default, the assets are pulled from Aquarius hosted by Ocean Protocol. To extend the existing features of the marketplace, developers can fork the marketplace.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace.md:8]: "provider a web interface" -> "provides a web interface"
"By default, the assets are pulled from Aquarius hosted by Ocean Protocol" this just doesn't sound right. It sounds like the OceanProtocol hosts the assets which is incorrect

File: content/tutorials/marketplace.md [hunk 10]
```diff
@@ -3,43 +3,19 @@ title: Set Up a Marketplace
...
 
-1. Develop the first cut of the app.
-1. Prepare some initial data assets.
-1. Deploy to production.
+By doing so, developers can:
 
-## Develop a First Cut of the App
+- Change the name of the marketplace.
+- Implement their own branding, style.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace.md:13]: "branding, style" -> "branding and style"

File: content/tutorials/marketplace.md [hunk 11]
```diff
@@ -3,43 +3,19 @@ title: Set Up a Marketplace
...
+- Change the source of asset information.
+- Change fee address.
```
Comment (AnaLoznianu) [inline content/tutorials/marketplace.md:15]: "Change fee address.". This is about custom fees? I think we can rephrase this to "Customise the fees" or smth like that

**Review by AnaLoznianu — CHANGES_REQUESTED** [review]

**Comment by AnaLoznianu** [comment]
@trentmc Can you please review this content?

File: content/tutorials/marketplace.md (hunk 9 above)
Comment (akshay-ap) [reply content/tutorials/marketplace.md:8]: how about ` By default, the metadata of the assets are pulled from Aquarius hosted by Ocean Protocol.`?

**Review by AnaLoznianu — APPROVED** [review]

File: content/tutorials/marketplace-fees.md (hunk 2 above)
Comment (DMats) [reply content/tutorials/marketplace-fees.md:14]: I posed the same question in the core team Slack. The conclusion was that datatoken is 1 word, base token is two words.

**Comment by DMats** [comment]
I think we need to specify for all fees:

1. Whether it's a percentage or an absolute value.
2. What is the minimum and maximum fee amounts allowed by the smart contracts

File: content/tutorials/marketplace-fees.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+- Consume Market Consumption Fee
+- Publisher Market Consumption Fee
+- Provider Consumption Fees
+- [Ocean Community Fee](#ocean-community-fee)
+
+### Ocean Community fee
+
+Ocean's smart contract collects the **Ocean Community fee** during a swap operation. These fees are reinvested in community projects via Ocean DAO and other initiatives. Currently, this fee is set to 0 (as long as either Ocean or H20 are used as the base token in the market) in the smart contract but can be updated by the Ocean Protocol foundation.
+
+Whereas a fee of 0.1% is applicable on all swaps using another token as the base token.
```
Comment (DMats) [inline content/tutorials/marketplace-fees.md:45]: OPC fees described in this paragraph are different from the amounts specified in table.

Here says `0%` for approved base tokens, `0.1%` for non-approved base tokens.
Table says `0.1%` for approved base tokens, `0.2%` for non-approved base tokens.

**Review by DMats — CHANGES_REQUESTED** [review]

File: content/tutorials/marketplace-fees.md (hunk 12 above)
Comment (AnaLoznianu) [reply content/tutorials/marketplace-fees.md]: Good catch @DMats. Changed the content now to reflect the table. o.1% and 0.2%

File: content/tutorials/marketplace-fees.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+Ocean's smart contract collects the **Ocean Community fee** during a swap operation. These fees are reinvested in community projects via Ocean DAO and other initiatives. Currently, this fee is set to 0.1% (as long as either Ocean or H20 are used as the base token in the market) in the smart contract but can be updated by the Ocean Protocol foundation.
+
+Whereas a fee of 0.2% is applicable on all swaps using another token as the base token.
+
+### Provider fee
+
+Provider is a component of Ocean Protocol's ecosystem which facilitates data consumption, starts compute jobs, encrypts DDOs, and decrypts DDOs. Provider also validates if the user can access a particular data asset or service. To know more about Provider, click [here](https://github.com/oceanprotocol/provider).
+
+Provider fees are paid to the individual or organization running their Provider instance whenever a user consumes a data asset or swaps a datatoken for a base token. These fees can be set to an absolute amount, and the receiver can also decide which token to receive the fees in - they don't have to be in the same currency used in the consuming market.
```
Comment (DMats) [inline content/tutorials/marketplace-fees.md:51]: The first sentence is incorrect - provider fees are not charged for swaps. 

Consider rewording:
> Provider fees are paid to the individual or organization running the Provider instance when the user orders an asset. These fees can be set to an absolute amount, not as a percentage. The provider can also specify which token the fees must be paid in - they don't have to be the same token used in the consuming market.

File: content/tutorials/marketplace-fees.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+
+Provider fee can also be used to charge for computing resources. Based on the compute resources needed to run an algorithm in the Compute-to-Data environment, a consumer can choose the amount to pay according to their needs.
```
Comment (DMats) [inline content/tutorials/marketplace-fees.md:53]: Consider making Provider fees plural here - to match the convention used throughout the other sections.

File: content/tutorials/marketplace-fees.md [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+
+This fee incentivizes individuals and organizations to run their provider instances and charge consumers according to resource usage.
```
Comment (DMats) [inline content/tutorials/marketplace-fees.md:55]: Consider making fees plural here - to match the convention used throughout the other sections.

**Review by DMats — CHANGES_REQUESTED** [review]

**Comment by AnaLoznianu** [comment]
@DMats Thanks for your feedback! We've updated the fees document with your suggestions

cc @alexcos20 @trentmc

**Review by AnaLoznianu — APPROVED** [review]

**Review by DMats — APPROVED** [review]
Great work!
